src/components/modules/banners/ausMoment/components/AusBannerTicker.tsx

```diff
@@ -0,0 +1,237 @@
+    }
+
+    const percentage = (total / end) * 100 - 100;
+
+    return `translate3d(${percentage >= 0 ? 0 : percentage}%, 0, 0)`;
+};
+
+const filledProgressStyles = (
+    end: number,
+    runningTotal: number,
+    total: number,
+    colour: string,
+): SerializedStyles =>
+    css`
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        transform: ${progressBarTransform(end, runningTotal, total)};
+        transition: transform 3s cubic-bezier(0.25, 0.55, 0.2, 0.85);
+        background-color: ${colour};
+    `;
+
+const goalContainerStyles = css`
+    position: absolute;
+    right: 0;
+    bottom: ${progressBarHeight + 5}px;
+    text-align: right;
+`;
+
+const goalMarkerStyles = (transform: string): SerializedStyles => css`
+    border-right: 2px solid ${palette.neutral[7]};
+    content: ' ';
+    display: block;
+    height: 12px;
+    margin-top: -2px;
+    transform: ${transform};
+`;
+
+// -- types -- //
+
+type MarkerProps = {
+    goal: number;
+    end: number;
+};
+
+type AusBannerTickerProps = {
+    settings: TickerSettings;
+    accentColour: string;
+    tickerData: TickerData;
+};
+
+const numberStyles = css`
+    ${headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+    color: #04ffff;
+    font-size: 36px;
+    background-color: 'black';
+    text-align: center;
+`;
+const textStyles = css`
+    color: #ffffff;
+    ${body.medium({ fontStyle: 'italic' })}
+    margin-top: -${space[1]}px;
+    text-align: center;
+`;
+
+const currentNumberContainerStyles = css`
+    background-color: #052962;
+    border-radius: 27px;
+    padding: 5px 15px;
+`;
+
+// -- components -- //
+
+const Marker: React.FC<MarkerProps> = ({ goal, end }: MarkerProps) => {
+    if (end > goal) {
+        const markerTranslate = (goal / end) * 100 - 100;
+        const markerTransform = `translate3d(${markerTranslate}%, 0, 0)`;
+
+        return <div css={goalMarkerStyles(markerTransform)} />;
+    } else {
+        return null;
+    }
+};
+
+interface CurrentSupporterNumberProps {
+    tickerData: TickerData;
+}
+
+export const CurrentSupporterNumber: React.FC<CurrentSupporterNumberProps> = ({
+    tickerData,
+}: CurrentSupporterNumberProps) => {
+    return (
+        <div css={currentNumberContainerStyles}>
+            <div css={numberStyles}>{tickerData.total.toLocaleString('en-US')}</div>
+            <div css={textStyles}>current supporters</div>
+        </div>
+    );
+};
+
+interface GoalSupporterNumber {
+    goal: number;
+}
+export const GoalSupporterNumber: React.FC<GoalSupporterNumber> = ({
+    goal,
+}: GoalSupporterNumber) => {
+    const ourGoal = goal.toLocaleString();
+    return (
+        <div css={currentNumberContainerStyles}>
+            <div css={numberStyles}>{ourGoal}</div>
+            <div css={textStyles}>Our goal</div>
+        </div>
+    );
+};
+
+export const AusBannerTicker: React.FC<AusBannerTickerProps> = ({
```

Comment:
(something to consider later)
@TomPretty do you remember if we used to have a ticker component for banner? Ideally epic + banner would share...

Reply:
We have a ticker for the template banner. I can't now remember why we did it as a separate ticker instead of reusing the epic one :/

Reply:
but agreed, we now have 3 tickers that are really all just the same